core/db/migrate/20160101010000_solidus_one_four.rb

```diff
@@ -0,0 +1,1153 @@
+class SolidusOneFour < ActiveRecord::Migration[5.0]
+  def up
+    # This migration is just a compressed version of all the previous
+    # migrations for spree_core. Do not run it if one of the core tables
+    # already exists. Assume the best.
+    return if table_exists?(:spree_addresses)
+
+    # this table should not technically exist in the database (as its provided by auth_devise),
+    # but spree_api depends on it existing. This defininition comes from solidus_auth_devise's first migration,
+    # and creates a table equivolent to it
+    create_table "spree_users", force: true do |t|
```

Comment:
Can't we fix `solidus_api` instead? It should not rely on this table.

Reply:
Discussed IRL, not reasonable to do with this PR (which doesn't change behavior). We'll need to change behavior to fix it elsewhere.

Reply:
👌🏻